cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_cov.cpp

```diff
@@ -41,10 +41,10 @@ static train_result call_daal_kernel(const context_cpu& ctx,
     const int64_t component_count = desc.get_component_count();
 
     auto arr_data = row_accessor<const Float>{ data }.pull();
-    array<Float> arr_eigvec{ column_count * component_count };
-    array<Float> arr_eigval{ 1 * component_count };
-    array<Float> arr_means{ 1 * component_count };
-    array<Float> arr_vars{ 1 * component_count };
+    auto arr_eigvec = array<Float>::zeros(column_count * component_count);
```

Comment:
Why need to spend time for initializing of array? when will we rewrite the contents anyway? This can take a lot of time in some algorithms. Especially if the filling is in sequential mode.

Reply:
fixed